credentials/apps/verifiable_credentials/issuance/models.py

```diff
@@ -66,6 +66,10 @@ class IssuanceLine(TimeStampedModel):
         help_text=_("Keeps track on a corresponding user credential's status"),
     )
 
+    class Meta:
+        ordering = ("created",)
+        unique_together = ("issuer_id", "status_index")
```

Comment:
Can you please comment on why it's required to make these two unique_together?
Is there a separate status list for every issuer config?

Reply:
Yepp. Since each Status List itself is an issued verifiable credential (it has its issuer, and this is the same one which produces and proves related VCs). So we have different Status Lists maintained by each issuer (a single system one in current configuration). And it is the [matter of verification steps](https://github.com/digitalcredentials/vc-status-list/blob/274e3fac4e39848a9d312214a4edc0635be0f28e/lib/index.js#L222) within the LCWallet as well.